src/pydantic2linkml/cli/__init__.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+
+import typer
+from linkml_runtime.dumpers import yaml_dumper
+
+from pydantic2linkml.gen_linkml import LinkmlGenerator
+from pydantic2linkml.tools import fetch_defs, get_all_modules
+
+app = typer.Typer()
+
+
+@app.command()
+def main(
+    module_names: list[str],
+    output_file: Path = None,
```

Comment:
is there a way in typer to assign short options too, like `-o` here?

Reply:
Yes and short option names are added.